src/stories/Button.stories.tsx

```diff
@@ -1,13 +1,15 @@
-import React from 'react';
-import { Story, Meta } from '@storybook/react';
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable react/jsx-props-no-spreading */
+import React from "react";
+import { Story, Meta } from "@storybook/react";
 
-import { Button, ButtonProps } from './Button';
+import { Button, ButtonProps } from "./Button";
 
 export default {
-  title: 'Example/Button',
+  title: "Example/Button",
   component: Button,
   argTypes: {
-    backgroundColor: { control: 'color' },
+    backgroundColor: { control: "color" },
```

Comment:
Having a background color as prop is giving a lot of freedom in terms of design. Usually there are only a few designs which **should** be possible :)